py3-azure-core.yaml

```diff
@@ -0,0 +1,47 @@
+# Generated from https://pypi.org/project/azure-core/
+package:
+  name: py3-azure-core
+  version: 1.29.6
+  epoch: 0
+  description: Microsoft Azure Core Library for Python
+  copyright:
+    - license: MIT License
```

Comment:
Think this should just be 'MIT'

Reply:
Cool, will fix it, thanks for noticing it